src/ssh-compute.ts

```
@@ -0,0 +1,214 @@
+    const sshKeyExpireAfter = core.getInput('ssh_key_expire_after');
+    const sshArgs = core.getInput('ssh_args');
+    let command = core.getInput('command');
+    const entrypoint = core.getInput('entrypoint');
+    const credentials = core.getInput('credentials');
+    let projectId = core.getInput('project_id');
+    let gcloudVersion = core.getInput('gcloud_version');
+    const generateSshKeys = core.getInput('generate_ssh_keys');
+  
+    // Flags
+    const internalIp = core.getInput('internal_ip');
+    const tunnelThroughIap = core.getInput('tunnel_through_iap');
+    const flags = core.getInput('flags');
+    const installBeta = false; // Flag for installing gcloud beta components
+    let cmd;
+
+    if (command && entrypoint) {
+      throw new Error(
+        'Both `command` and `entrypoint` inputs set - Please select one.',
+      );
+    }
+  
+    if (internalIp && tunnelThroughIap) {
+      throw new Error(
+        'Both `internal_ip` and `tunnel_through_iap` inputs set - Please select one.',
+      );
+    }
+
+    if (user) {
+      instanceName = `${user}@${instanceName}`;
+    }
+
+    cmd = [
+      'compute',
+      'ssh',
+      instanceName,
+      '--zone',
+      zone,
+      '--quiet', // we need to ignore promts from console
+    ];
+
+    if (container) {
+      cmd.push('--container', container);
+    }
+
+    if (sshKeyFilePath) {
+      cmd.push('--ssh-key-file', sshKeyFilePath);
+    }
+
+    if (sshKeyExpireAfter) {
+      cmd.push('--ssh-key-expire-after', sshKeyExpireAfter);
+    }
+
+    if (tunnelThroughIap) {
+      cmd.push('--tunnel-through-iap');
+    }
+
+    if (internalIp) {
+      cmd.push('--internal-ip');
+    }
+
+    if (flags) {
+      const flagList = parseFlags(flags);
+      if (flagList) cmd = cmd.concat(flagList);
+    }
+
+    if (entrypoint) {
+      if (!fs.existsSync(entrypoint)) {
+        core.error(`${entrypoint} does not exist.`);
```

We can just throw the error, I dont think this adds additional value.